config.local/arm64/config.sonic-aspeed

```diff
@@ -0,0 +1,105 @@
+CONFIG_ARCH_ASPEED=y
+CONFIG_NET_VENDOR_FARADAY=y
+CONFIG_FTGMAC100=y
+CONFIG_UNICODE=y
+CONFIG_BLOCK=y
+CONFIG_LBDAF=y
+CONFIG_FS_MBCACHE=y
+CONFIG_JBD2=y
+CONFIG_EXT4_FS=y
+CONFIG_EXT4_USE_FOR_EXT2=y
+CONFIG_EXT4_FS_POSIX_ACL=y
+CONFIG_EXT4_FS_SECURITY=y
+CONFIG_MCTP=y
+CONFIG_SERIAL_8250_ASPEED_VUART=y
+CONFIG_SERIAL_8250_ASPEED=y
+CONFIG_SERIAL_8250_NR_UARTS=24
+CONFIG_SERIAL_8250_RUNTIME_UARTS=24
+CONFIG_CAN_ASPEED=m
+CONFIG_MDIO_DEVICE=m
+CONFIG_MDIO_BUS=m
+CONFIG_MDIO_ASPEED=m
+CONFIG_ASPEED_KCS_IPMI_BMC=m
+CONFIG_ASPEED_BT_IPMI_BMC=m
+CONFIG_SPI_ASPEED_SMC=m
+CONFIG_I2C_ASPEED=m
+CONFIG_SPI_ASPEED_TXRX=y
+CONFIG_PINCTRL_ASPEED=y
+CONFIG_PINCTRL_ASPEED_G7=y
+CONFIG_GPIO_ASPEED=y
+CONFIG_GPIO_ASPEED_SGPIO=y
+CONFIG_GPIO_ASPEED_LTPI=y
+CONFIG_SENSORS_ASPEED=y
+CONFIG_SENSORS_ASPEED_G6=m
+CONFIG_ASPEED_WATCHDOG=y
+CONFIG_MEDIA_SUPPORT=y
+CONFIG_VIDEO_DEV=y
+CONFIG_DVB_CORE=y
+CONFIG_VIDEO_V4L2_I2C=y
+CONFIG_VIDEO_V4L2_SUBDEV_API=y
+CONFIG_MEDIA_CONTROLLER=y
+CONFIG_V4L2_FWNODE=y
+CONFIG_V4L2_ASYNC=y
+CONFIG_DVB_NET=y
+CONFIG_DVB_MAX_ADAPTERS=16
+CONFIG_DVB_DYNAMIC_MINORS=y
+CONFIG_MEDIA_PLATFORM_DRIVERS=y
+CONFIG_V4L_PLATFORM_DRIVERS=y
+CONFIG_VIDEO_ASPEED=y
+CONFIG_VIDEOBUF2_CORE=y
+CONFIG_VIDEOBUF2_V4L2=y
+CONFIG_VIDEOBUF2_MEMOPS=y
+CONFIG_VIDEOBUF2_DMA_CONTIG=y
+CONFIG_VIDEOBUF2_VMALLOC=m
```

Comment:
Can the video and DVB-related configs be removed?

Reply:
This is fine for now, but later, I'd like to see those configs be removed; if some Aspeed code doesn't compile because of these missing configs, that suggests their code needs fixing.